src/app/pages/send/send-crypto-asset-form/form/stacks-sip10/stacks-sip10-fungible-token-send-form.tsx

```diff
@@ -12,28 +12,42 @@ import { SendMaxButton } from '../../components/send-max-button';
 import { StacksCommonSendForm } from '../stacks/stacks-common-send-form';
 import { useSip10SendForm } from './use-sip10-send-form';
 
-export function StacksSip10FungibleTokenSendForm() {
+export function StacksSip10FungibleTokenSendFormSentinel() {
```

Comment:
not sure with naming here

Reply:
Maybe the loader pattern/naming would work here, similar to home component?

Where loader takes care of the navigation when undefined, and passes to child component like 

https://github.com/hirosystems/wallet/blob/af69fd665d1ddb718f6ee949dc068a5d45b57a7f/src/app/pages/home/home.loader.tsx#L5-L13

pseudo ala

```ts
export function StacksSip10FungibleTokenSendForm() {
  return (
    <StacksSip10FtSendFormLoader>
      {({ contractId, symbol }) => // the rest of the component }
    </StacksSip10FtSendFormLoader>
  )
}
```